pkg/kubelet/server/stats/handler.go

```diff
@@ -78,6 +78,8 @@ type StatsProvider interface {
 	ListVolumesForPod(podUID types.UID) (map[string]volume.Volume, bool)
 	// GetPods returns the specs of all the pods running on this node.
 	GetPods() []*v1.Pod
+	// GetPodCgroupRoot returns the cgroup containing all pods
+	GetPodCgroupRoot() string
```

Comment:
can you note that this is the literal cgroupfs value?

Reply:
done